README.md

```diff
@@ -26,6 +26,7 @@ The Rust build system uses a Python script called `x.py` to build the compiler,
 which manages the bootstrapping process. More information about it can be found
 by running `./x.py --help` or reading the [rustc dev guide][rustcguidebuild].
 
+[rustc-dev-guide]: https://rustc-dev-guide.rust-lang.org
```

Comment:
Wait, what is this doing? There's no `rustc-dev-guide` link anywhere.

Reply:
Wow, that's a long way down 😆 can you move this link down there so it's clear what it's for?

Reply:
Sure. I just put it there because that's where the other dev guide links were :)

Reply:
Okay, I moved it!